src/EffectID.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
-enum class EffectID : Int32
+// tolua_begin
+
+enum EffectID
```

Comment:
This is not a good change. The `: Int32` specifier was there on purpose, so that the enum cannot be implicitly-converted from other types. Since ToLua doesn't support this syntax, you should use a slightly different approach:
```cpp
#ifndef TOLUA_EXPORT_PARSER
  enum class EffectID: Int32
#else
  enum EffectID  // tolua_export
#endif
// tolua_begin
{
  ...
};
```